tests/unit/test_protocol_errors.py

```diff
@@ -22,6 +22,7 @@ class TestFrontendErrors(unittest.TestCase):
     """
     @classmethod
     def setUpClass(cls):
+        reload(frontend)
```

Comment:
modules are only loaded once by default, so if we don't reload then the flask app is not reset for the new test.